scripts/compile_libs.sh

```diff
@@ -13,8 +13,15 @@ tar -zxvf solvers.tgz
 # Compile ASL, warnings about files existing seem to be okay
 
 cd solvers
-./configure
-make
+
+# adowling laptop
+./configure CC=gcc-10 F77=gfortran-10
```

Comment:
@eslickj Any advice on how to streamline this? Set CC and F77 flags at the top of the script?